applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/eigensolver_strategy.hpp

```diff
@@ -672,19 +672,20 @@ class EigensolverStrategy
                 rNodeEigenvectors.resize(NumEigenvalues,NumNodeDofs,false);
             }
 
-            // the jth column index of EIGENVECTOR_MATRIX corresponds to the jth nodal dof. therefore,
-            // the dof ordering must not change.
-            if (NodeDofs.IsSorted() == false)
-            {
-                NodeDofs.Sort();
-            }
+            // TO BE VERIFIED!! In the current implmentation of Dofs there are nor reordered and only pushec back. 
+            // // the jth column index of EIGENVECTOR_MATRIX corresponds to the jth nodal dof. therefore,
+            // // the dof ordering must not change.
+            // if (NodeDofs.IsSorted() == false)
+            // {
+            //     NodeDofs.Sort();
+            // }
```

Comment:
@msandre I think you initially wrote this, can you please have a look?

Reply:
I believe it is ok as long as the dof ordering is fixed. You could use std::sort if you don't want to change the behavior.